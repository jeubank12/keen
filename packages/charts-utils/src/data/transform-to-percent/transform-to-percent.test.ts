import transformToPercent from './transform-to-percent';

test('transform series values to percents', () => {
  const data = [
    { day: 'Monday', woman: 10, man: 5 },
    { day: 'Tuesday', woman: 20, man: 20 },
  ];

  const result = transformToPercent(data, ['man', 'woman']);

  expect(result).toMatchSnapshot();
});
