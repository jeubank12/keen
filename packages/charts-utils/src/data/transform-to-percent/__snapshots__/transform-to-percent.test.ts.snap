// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform series values to percents 1`] = `
Array [
  Object {
    "day": "Monday",
    "man": 33.33333333333333,
    "woman": 66.66666666666666,
  },
  Object {
    "day": "Tuesday",
    "man": 50,
    "woman": 50,
  },
]
`;
